docs/object.md

```diff
@@ -1280,7 +1280,7 @@ newDecks[2].deal(1)
 
 ####deal(...)
 
-[<span class="ret boo"></span>](types.md)&nbsp;Deals Objects to hand zones. Will deal from decks/bags/stacks as well as individual items. If dealing an individual item to a hand zone, it is a good idea to make sure that its [Member Variable](#member-variables) for `use_hands` is `true`.
+[<span class="ret obj"></span>](types.md)&nbsp;Deals Objects to hand zones. Will deal from decks/bags/stacks as well as individual items. If dealing an individual item to a hand zone, it is a good idea to make sure that its [Member Variable](#member-variables) for `use_hands` is `true`.
```

Comment:
Other changes look good, however this is a mistake. `deal` returns a boolean, not an object.

It would certainly be an API improvement if there was a way to easily get access to the dealt objects. However, returning a single object be enough, as this method can deal multiple cards.

Reply:
It's probably worth noting that not only does this method return a boolean, it's _always_ `true`.

Reply:
true, my bad, changed it